### PR TITLE
Hotwire with redis

### PR DESCRIPTION
**Pull request by JulianPasquale**
Add **Hotwire with Redis** section to `README` with a basic configuration for Action Cable with Redis.

### CODE REVIEW

**Review by martinjaimem — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -66,6 +66,22 @@ This template comes with:
...
+
+Action Cable relies on `redis` as [subscription adapter](https://guides.rubyonrails.org/action_cable_overview.html#subscription-adapter) for production environment.
+
+With just a little configuration you can make it work.
+
+```yaml
+# config/cable.yml
+
+production:
+  adapter: redis
```
Comment (santib) [inline README.md:81]: not sure but I guess redis should be a symbol here?
```suggestion
  adapter: :redis
```

File: config/cable.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 development:
   adapter: redis
```
Comment (santib) [inline config/cable.yml:2]: maybe we can use `inline` ?

**Review by santib — APPROVED** [review]

File: config/cable.yml (hunk 2 above)
Comment (JulianPasquale) [reply config/cable.yml:2]: According to [Action Cable docs](https://guides.rubyonrails.org/action_cable_overview.html#dependencies), _The default adapter in new Rails applications is the asynchronous (async) adapter_.
WDYT if we use `async` instead? Assuming Hotwire will still working 😅

Comment (santib) [reply config/cable.yml:2]: yes sorry for the confusion, I meant that one haha 😂

**Pull request merged**